sdk/ai/ai-projects/samples-dev/agents/agentsBasics.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to use basic agent operations from the Azure Agents service using a synchronous client.
+ *
+ * @summary demonstrates how to use basic agent operations.
+ *
```

Comment:
no need to mention about synchronous client , Python has synchronous and async client its not applicable to JS

Reply:
Removed sync/async information